ddtrace/opentelemetry/tracer.go

```diff
@@ -95,3 +95,5 @@ type noopOteltracer struct{}
 func (n *noopOteltracer) Start(_ context.Context, _ string, _ ...oteltrace.SpanStartOption) (context.Context, oteltrace.Span) {
 	return nil, nil
 }
+
+var telemetryTags = []string{"otel"}
```

Comment:
super duper nit, can we put this at the top with the other decls?

Reply:
Yeah sure, just figured it was the most out of the way at the bottom. But don't feel strongly